lua/pac3/core/client/parts/proxy.lua

```diff
@@ -908,6 +960,8 @@ function PART:RunExpression(ExpressionFunc)
 	return pcall(ExpressionFunc)
 end
 
+local oldclass = ""

```

Comment:
?

Reply:
Used to only update the expression if the parent becomes a different class?